DependencyInjection/Configuration.php

```diff
@@ -41,6 +41,10 @@ public function getConfigTreeBuilder()
                             ->scalarNode('protocol')->defaultValue('Thrift\Protocol\TBinaryProtocolAccelerated')->end()
                             ->scalarNode('transport')->defaultValue('Thrift\Transport\TBufferedTransport')->end()
                             ->booleanNode('server')->defaultFalse()->end()
+                            ->arrayNode('includeDirs')
+                                ->prototype('scalar')
```

Comment:
I can't understand what exactly you mean.
Isn't it the default way to describe configuration when we expect yaml structures to be like:
```yaml
includeDirs:
  - dir1
  - dir2
```
?

With this configuration if I pass different yamls, i get these "responses":
.1. includeDirs is scalar
```yaml
includeDirs: dir1
```
```
Invalid type for path "overblog_thrift_bundle.services.thrift.includeDirs". Expected array, but got string
```
.2. includeDirs is array of scalars
```yaml
includeDirs:
  - dir1
  - dir2
```
```
      ["includeDirs"]=>
      array(2) {
        [0]=>
        string(4) "dir1"
        [1]=>
        string(4) "dir2"
      }
```
.3. includeDirs is array of arrays
```yaml
includeDirs:
  - [dir1,dir2]
  - [dir3,dir4]
```
```
Invalid type for path "overblog_thrift_bundle.services.thrift.includeDirs.0". Expected scalar, but got array.
```
So everything looks ok, isn't it?